src/consensus/vote.cpp

```diff
@@ -785,15 +790,14 @@ void NextVotesForPreviousRound::updateNextVotes(std::vector<Vote> const& next_vo
 
 // Assumption is that all synced votes are in next voting phase, in the same round.
 // Valid voted values have maximum 2 block hash, NULL_BLOCK_HASH and a non NULL_BLOCK_HASH
-void NextVotesForPreviousRound::updateWithSyncedVotes(std::vector<Vote> const& next_votes,
-                                                      size_t const pbft_2t_plus_1) {
-  // TODO: need do vote verificaton
+void NextVotesForPreviousRound::updateWithSyncedVotes(std::vector<Vote>& next_votes, size_t const pbft_2t_plus_1) {
```

Comment:
why removed const from next_votes ?

Reply:
Because verifyVrfSortition() is not const
https://github.com/Taraxa-project/taraxa-node/blob/next_votes_bundle_verification/src/consensus/vote.cpp#L918